data/json/items/tool/cooking.json

```diff
@@ -1051,5 +1051,89 @@
     "pocket_data": [ { "pocket_type": "MAGAZINE_WELL", "flag_restriction": [ "GAS_TANK" ], "default_magazine": "small_propane_tank" } ],
     "charges_per_use": 1,
     "use_action": [ "HOTPLATE", "HEAT_FOOD" ]
+  },
+  {
+    "id": "blanching_acorns_clay",
+    "type": "GENERIC",
+    "category": "food",
+    "name": { "str_sp": "pot of blanching acorns" },
```

Comment:
```suggestion
    "name": { "str": "pot of blanching acorns", "str_pl": "pots of blanching acorns" },
```

Reply:
thanks, updating all of them. Couldn't remember what the strings were.